frontend/src/modules/2DViewer/layers/implementations/layers/DrilledWellTrajectoriesLayer/types.ts

```diff
@@ -0,0 +1,8 @@
+import { WellboreHeader_api } from "@api";
+import { EnsembleIdent } from "@framework/EnsembleIdent";
+import { SettingType } from "@modules/2DViewer/layers/implementations/settings/settingsTypes";
+
+export type DrilledWellTrajectoriesSettings = {
```

Comment:
Can this "implement" or be type of `Settings` in `interfaces.ts`?

Reply:
A type can neither implement an interface (only a class can) nor be of another type. As far a I know, the only way to get something like this done would involve some ugly generics making the code less readable. If your intention is to validate that the settings type provided is valid, then this will happen when applying it to a `SettingsContext` / `Layer`. I don't see a nice/clean method to check it up-front. But maybe there is one that I don't know of.

Reply:
Then we leave it as it is 👍